python_modules/dagster/dagster_tests/daemon_tests/test_monitoring_daemon.py

```diff
@@ -135,34 +141,50 @@ def test_monitor_starting(instance, logger):
         instance.get_run_by_id(run.run_id),
         logger,
     )
-    assert instance.get_run_by_id(run.run_id).status == DagsterRunStatus.FAILURE
+    run = instance.get_run_by_id(run.run_id)
+    assert run
+    assert run.status == DagsterRunStatus.FAILURE
 
 
-def test_monitor_started(instance, workspace, logger):
+def test_monitor_started(
+    instance: DagsterInstance, workspace_context: WorkspaceProcessContext, logger: Logger
+):
     run = create_run_for_test(instance, pipeline_name="foo", status=DagsterRunStatus.STARTED)
+    workspace = workspace_context.create_request_context()
```

Comment:
seems like an `IWorkspaceContext` instead of an `IWorkspace` was being used incorrectly here.